<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>615</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Han Sans</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>异梦 ver 2.0.1 重构版本 build 220405</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Pictures/D:/Genshin Impact International/Genshin Impact/Genshin Impact game/ScreenShot/20220320123857.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans Heavy</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>异梦</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>by @BootLoaderANK，使用Qt重构</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>此程序提供原神服务器互转功能而无需重新下载约30GB的文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans Heavy</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>数据包</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>要替换数据包，请转到「文件」选项卡下的「更新资源包」</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据包对应版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Datapack_Version_Display">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/target_version</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Progress">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据包方向：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Datapack_Direction_Display">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/target_direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans Heavy</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Installer_Dir">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>481</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>启动器安装目录：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Game_Dir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Genshin Impact Game目录：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Convert_Start">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(33, 150, 243);</string>
    </property>
    <property name="text">
     <string>开始转换</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>479</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最后一条日志将会显示在这里</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update_Datapack_Info">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>270</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Han Sans ExtraLight</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>更新/加载数据包信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>开发</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>更新资源包</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>测试模式</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>沙盒</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>许可</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>更新日志</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>关于 异梦</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
